src/app/landing/LandingPage.css

```diff
@@ -41,11 +41,12 @@
   flex-direction: column;
   justify-content: space-between;
   align-items: center;
-  background: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.6)),url('/img/landing_page/top1.jpg') no-repeat center center fixed;
+  background: linear-gradient(to bottom, rgba(0, 0, 0, 0), rgba(0, 0, 0, 0.6)),url('/img/landing_page/top1_desktop.jpg') no-repeat center center fixed;
   background-position: 30% 0%;
   background-size: cover;
   color: #FFFFFF;
   padding: 15vh 0px 15vh 0px;
+  overflow: hidden;
```

Comment:
Not a problem, but just out of curiosity, how come this was needed? Aren't all the heights in vh units, meaning they shouldn't be able to overflow on any screen size?

Reply:
Background image overflows on very small devices (like iPhone5/SE) so this line resolves that issue, open to suggestions for improvement!